react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isToggled: false,
+      className: 'notToggled',
+      container: 'container notToggled',
+      onOrOff: 'OFF'
+    };
+    this.handleToggle = this.handleToggle.bind(this);
+  }
+
+  handleToggle() {
+    const isToggled = this.state.isToggled;
+    if (isToggled) {
+      this.setState({ isToggled: false });
+      this.setState({ className: 'notToggled' });
```

Comment:
for future reference, setState takes an object as an argument, so if you ever have to update multiple properties at once, you can just pass it an object with each of the properties you want to update.